pkg/asset/installconfig/gcp/validation.go

```diff
@@ -248,3 +248,19 @@ func ValidateEnabledServices(ctx context.Context, client API, project string) er
 	}
 	return nil
 }
+
+// ValidateProjectRegion determines whether the region is valid for the project
+func ValidateProjectRegion(ctx context.Context, client API, project string, region string) error {
+	validRegions, err := client.GetProjectRegions(ctx, project)
+	if err != nil {
+		return err
+	}
+
+	for r := range validRegions {
+		if region == validRegions[r] {
+			return nil
+		}
+	}
```

Comment:
Seems like another place where having `GetProjectRegions` return a map would be good.